tests/integration/block_createBlock_api_test.go

```diff
@@ -55,7 +55,7 @@ func (s *BlockImageCreateSuite) SetupSuite() {
 
 	s.installer = installer.NewK8sRookhelper(kh.Clientset, s.T)
 
-	err = s.installer.InstallRookOnK8s("rook")
+	err = s.installer.InstallRookOnK8s("rook", "")
```

Comment:
it would be good to have another fund called `InstallRookOnK8sWithHostPath` so all the callers don't have to pass an empty host path here